examples/nlp/text_classification/text_classification_with_bert.py

```diff
@@ -113,14 +129,15 @@ def create_pipeline(num_samples=-1, batch_size=32, num_gpus=1, local_rank=0, mod
     data_file = f'{data_desc.data_dir}/{mode}.tsv'
     shuffle = args.shuffle_data if mode == 'train' else False
 
-    data_layer = nemo.collections.nlp.nm.data_layers.text_classification_datalayer.BertTextClassificationDataLayer(
+    data_layer = nemo_nlp.nm.data_layers.BertTextClassificationDataLayer(
         input_file=data_file,
         tokenizer=tokenizer,
         max_seq_length=args.max_seq_length,
         num_samples=num_samples,
         shuffle=shuffle,
         batch_size=batch_size,
-        use_cache=args.use_cache,
+        use_cache=not args.no_data_cache,
+        model_name=args.pretrained_model_name,
```

Comment:
Why does data_layer need model_name?

Reply:
different model use different tokenizers, so we need to add model name to the cache file name to distinguish preprocess datatsets

Reply:
could you change all datalayers to have this feature for consistency?